common/tests/ontology_query_interface_unit_tests.py

```diff
@@ -0,0 +1,187 @@
+import unittest
+import os
+from mas_knowledge_utils.ontology_query_interface import OntologyQueryInterface
+
+class ontology_query_interface_test(unittest.TestCase):
+    '''Implements unit tests for the OntologyQueryInterface APIs
+    using the sample.owl ontology
+
+    @author Sushant Chavan
+    @contact sushant.chavan@smail.inf.h-brs.de
+
+    '''
+
+    def setUp(self):
+        # Get directory paths
+        script_dir = os.path.abspath(os.path.dirname(__file__))
+        ontology_dir = os.path.join(os.path.dirname(script_dir), "ontology")
+        assertions_dir = os.path.join(os.path.dirname(script_dir), "assertions")
```

Comment:
This variable is not used anywhere; can you please remove it?

Reply:
Removed the variable in the new commit.